src/main/avro/it/bitrock/dvs/model/AirlineRaw.avdl

```diff
@@ -2,15 +2,15 @@
 protocol AirportRawProtocol {
 
   record AirlineRaw {
-    long airlineId;
-    string nameAirline;
-    string codeIataAirline;
-    string codeIcaoAirline;
+    long id;
+    string name;
+    string codeIata;
```

Comment:
maybe `iataCode`?

Reply:
👍 
I would  too use `iataCode` and `icaoCode`